File: push-openapi-net-demo/QueryAppPushStat.cs
using Aliyun.Acs.Core;
using Aliyun.Acs.Core.Exceptions;
using Aliyun.Acs.Core.Profile;
using Aliyun.Acs.Push.Model.V20150827;
using System;

namespace AlibabaCloud
{
    class QueryAppPushStat
    {
        static void Main()
        {
            IClientProfile clientProfile = DefaultProfile.GetProfile("cn-hangzhou", "<your access key id>", "<your access key secret>");
	    DefaultAcsClient client = new DefaultAcsClient(clientProfile);
       	    QueryAppPushStatRequest request = new QueryAppPushStatRequest();
            request.AppKey = <Your AppKey>;

            request.Granularity = "DAY";
            String startTime = DateTime.UtcNow.AddDays(-7).ToString("yyyy-MM-ddTHH\\:mm\\:ssZ");
            String endTime = DateTime.UtcNow.ToString("yyyy-MM-ddTHH\\:mm\\:ssZ");
            request.StartTime = startTime;
            request.EndTime = endTime;

            try
            {
                QueryAppPushStatResponse response = client.GetAcsResponse(request);
                Console.WriteLine("RequestId:" + response.RequestId);
                foreach (QueryAppPushStatResponse.AppPushStat stat in response.AppPushStats)
                {
                    Console.WriteLine("MessageIdt:" + stat.Time);
                    Console.WriteLine("SentCountt:" + stat.SentCount);
                    Console.WriteLine("ReceivedCount:" + stat.ReceivedCount);
                }
                Console.ReadLine();
            }
            catch (ServerException e)
            {
                Console.WriteLine(e.ErrorCode);
                Console.WriteLine(e.ErrorMessage);
                Console.ReadLine();
            }
            catch (ClientException e)
            {
                Console.WriteLine(e.ErrorCode);
                Console.WriteLine(e.ErrorMessage);
                Console.ReadLine();
            }
        }

    }
}

File: push-openapi-php-demo/QueryAppPushStat.php
<?php

include_once '../aliyun-php-sdk-core/Config.php';
use \QueryAppPushStat\Request\V20150827 as QueryAppPushStat;

$accessKeyId = "";
$accessSecret = "";
$appKey = 123456;
$iClientProfile = DefaultProfile::getProfile("cn-hangzhou", $accessKeyId, $accessSecret);
$client = new DefaultAcsClient($iClientProfile);
$request = new QueryAppPushStat\QueryAppPushStatRequest();

$request->setAppKey($appKey);
$request->setGranularity("DAY");
$request.setStartTime(startTime);
$request.setEndTime(endTime);

print_r("\r\n");
print_r($response);

?>
